src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -90,6 +95,9 @@ public void setPivotMotorAngle(double angle) {
 
   @Override
   public void periodic() {
+
     // This method will be called once per scheduler run
+    SmartDashboard.putNumber("Intake/Roller Percent Output", getRollerPercentOutput());
+
```

Comment:
Now that we have pivot motor we should be printing out the pivot motor angle as well

Reply:
nuh uh